tests/activate_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+mod_env() {
+    DIR="$1"
+    OLD="$2"
+    cat "$OLD" |grep -wv -e PATH -e MANPATH -e PS1
+    cat - <<EOT
+_KERL_ACTIVE_DIR=$DIR
+_KERL_ERL_AFLAGS_SET=
+_KERL_MANPATH_REMOVABLE=$DIR/lib/erlang/man:$DIR/man
+_KERL_PATH_REMOVABLE=$DIR/bin
+_KERL_REBAR_CACHE_DIR_SET=
+_KERL_REBAR_PLT_DIR_SET=
+_KERL_SAVED_REBAR_CACHE_DIR=
+_KERL_SAVED_REBAR_PLT_DIR=
+ERL_AFLAGS=-kernel shell_history enabled
+MANPATH=$DIR/lib/erlang/man:$DIR/man:$MANPATH
+PS1=$PS1
+REBAR_CACHE_DIR=$DIR/.cache/rebar3
+REBAR_PLT_DIR=$DIR
+EOT
+    ERLCALLDIR=`\find "$DIR" -type d -path "*erl_interface*/bin" 2>/dev/null`
+    if [ -n "$ERLCALLDIR" ]; then
+        echo "_KERL_ERL_CALL_REMOVABLE=$ERLCALLDIR"
+        echo "PATH=$ERLCALLDIR:$DIR/bin:$PATH"
+    else
+        echo "_KERL_ERL_CALL_REMOVABLE="
+        echo "PATH=$DIR/bin:$PATH"
+    fi
+}
+
+test_it() {
+    ./kerl emit-activate "$1" "$1" "$2" >/tmp/activate.sh
```

Comment:
It would probably be better to name the variables at the start of the function instead of using $1, $2, ... I already have an [issue to do that](https://github.com/kerl/kerl/issues/453) in other scopes, but we should prevent new occurrences 😄